### PR TITLE
 badge html and CSS

### PR DESCRIPTION
**Pull request by aniketsingh13**
https://user-images.githubusercontent.com/85512784/153553850-928946a8-27f3-49a4-95b7-5f33401611f9.png

### CODE REVIEW

File: Styles/badge.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    height: 150px;
+    gap: .5rem;
+    box-shadow: 1px 2px 1px 1px var(--boxShadow-color);
+    border-radius: 4px;
+}
+
+.badge_variant {
+    position: relative;
+    width: 50px;
+    height: 50px;
```
Comment (rohanmathur91) [inline Styles/badge.css]: Use rem instead of px.

File: Styles/badge.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    bottom: 0;
+    right: 0;
+    position: absolute;
+}
+
+/* badge-text */
+.badges-number {
+    border: 1px solid var(--border-color);
+    width: 100%;
+    height: 150px;
```
Comment (rohanmathur91) [inline Styles/badge.css]: same as above

File: Styles/badge.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    border-radius: 4px;
+}
+
+.badge {
+    position: relative;
+}
+
+.badge-greenNumber{
+    width: 20px;
+    height: 20px;
```
Comment (rohanmathur91) [inline Styles/badge.css]: same as above

File: Styles/badge.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    background-color: var(--success-color);
+    bottom: 1.3rem;
+    left: 1.6rem;
+    position: absolute;
+    color: var(--offwhite);
+}
+.badge-rednumber{
+    width: 20px;
+    height: 20px;
```
Comment (rohanmathur91) [inline Styles/badge.css]: same as above

**Review by rohanmathur91 — APPROVED** [review]
Use rem where possible, rest looks good

File: Styles/badge.css (hunk 1 above)
Comment (aniketsingh13) [reply Styles/badge.css]: will change it into the rem

File: Components/badge.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+            <div class="main__container">
+                <h3 class="p-xs f-l font-xxl">Badge</h3>
+                <div class="badge_intro  p-xs">Badges are small circles, positioned either at top-right or bottom-right
+                    of the parent component</div>
+                <div class="badge_intro  p-xs">Profile Badge is used to show the current status of a user</div>
+                <div class="badge_type p-xs">It can be used with 3 variants : online | offline | dnd</div>
+                <div class="badge_image mt-l flex flex-center">
+                    <div class="badge_variant">
+                        <image class="badge-green br-s "
+                            src="../Assets/spiderman-into-spider-verse-5k-7b-2048x2048-1020x1020.jpg" alt="Avatar" />
```
Comment (hiteshverma27) [inline Components/badge.html:72]: Try to use a icon library to reduce file size

**Review by hiteshverma27 — APPROVED** [review]
Use rems as suggested by rohan and rest looks good

File: Components/badge.html (hunk 5 above)
Comment (aniketsingh13) [reply Components/badge.html:72]: thanks will do that

**Pull request merged**